packages/shuttle_websocket/shuttle_websocket.0.12.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Websocket support for HTTP/1.1 servers using Async"
+description:
+  "Shuttle_websocket is a companion library for shuttle_http that provides a HTTP service that performs websocket negotiation for HTTP/1.1 servers."
+maintainer: ["Anurag Soni <anurag@sonianurag.com>"]
+authors: ["Anurag Soni"]
+license: "MIT"
+tags: ["http-server" "websocket"]
+homepage: "https://github.com/anuragsoni/shuttle_http"
+bug-reports: "https://github.com/anuragsoni/shuttle_http/issues"
+depends: [
+  "dune" {>= "3.1"}
+  "shuttle_http" {= version}
+  "async_websocket"
```

Comment:
```suggestion
  "async_websocket" {>= "v0.17.0"}
```
Seems needed to avoid the current lower bound failure `async_websocket not found`

Reply:
Oh that makes sense! I'll commit the change here and then reproduce this upstream. Thank you!

Reply:
Upstreamed at https://github.com/anuragsoni/shuttle_http/commit/740f9c4c0a4647eb457643c1323a40a3481fb62c